filterable-file-log/serializer.lua

```diff
@@ -60,11 +64,46 @@ function _M.serialize(ngx, filters)
     service = ngx.ctx.service,
     api = ngx.ctx.api,
     consumer = ngx.ctx.authenticated_consumer,
+    jwt_claims = jwt_claims,
```

Comment:
This seems OK for now but two things to consider as we live and learn with this implementation:

1. Should this be embedded in the request object?
2. Should this be omitted if `jwt_claims` is nil?

Reply:
1. I'm not sure! Do you think it should? I don't really have any opinion on the shape of this log line.

2. It actually will be omitted in that case, since in lua, setting the value of a key in a table to `nil` removes it

Reply:
It occurs to me that in 2 you might have been making just that point. I think omitting it is fine since we're not going to be migrating old log entries to the new format, so any query will have to assume that this field might not be present.

Reply:
I was concerned that it would show up with an explicit `null` value. I'd prefer the current behavior of it being omitted. I'm not sure how I feel about embedding this in the request object. I would have to dig deeper into how the rest of the standard plugins treat that object. We can handle that when this is extracted into it's own plugin.